remotemonitoring/armtemplates/standard.json

```diff
@@ -498,4 +501,4 @@
             "value": "[listSecrets(variables('bingMapsName'), variables('bingMapsVersion')).QueryKey]"
```

Comment:
There are a bunch of output values that `reference` the IoT Hub (e.g. [see line 473](https://github.com/Azure/pcs-cli/pull/239/files#diff-0d5a330fd8bf17136fefca28a8f04322R473)). These will need some workaround too.